index.html

```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+
+<div class="block">
```

Comment:
- [ ] Kindly use semantic approach instead of using div.
* For example:
```suggestion
<section class="block">
```

Reply:
- [ ] It's awesome stating that that the developer should use semantic approach, but your response was not explanatory enough; though the example clarified it a bit. 😄 You should have stated clearly the reason(s) for using `section `over `div`.